LongestSubstring.java

```diff
@@ -0,0 +1,31 @@
+import java.util.HashSet;
+import java.util.Set;
+//TC - O(n)
+// SC - O(N) - N is length of resultant longest substring
```

Comment:
Resultant space does not count in complexity

Reply:
Yes, modified the SC to O(1)